### PR TITLE
Android tv input text support

### PR DESCRIPTION
**Pull request by marcinnowak2**
Android TV input text support

### CODE REVIEW

File: ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java [hunk 1]
```diff
@@ -787,8 +788,40 @@ public boolean onEditorAction(TextView v, int actionId, KeyEvent keyEvent) {
...
+              switch (actionId) {
+                case EditorInfo.IME_ACTION_PREVIOUS: {
+                      @SuppressLint("WrongConstant") View view = editText.focusSearch(View.FOCUS_BACKWARD);
+
+//                      if (view instanceof ReactEditText) {
+//                        view.requestFocus();
+//                        ((ReactEditText) view).showKeyboard();
+//                      } else {
+//                        if (view != null) {
+//                          view.requestFocus();
```
Comment (ndillon1) [inline ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java]: can we remove this commented code ?

Comment (marcinnowak2) [reply ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java]: Sure.

Comment (marcinnowak2) [reply ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java]: Done.

Comment (marcinnowak2) [reply ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java]: Sure.

**Pull request merged**